src/ParallelAlgorithms/DiscontinuousGalerkin/CollectDataForFluxes.hpp

```diff
@@ -134,6 +134,22 @@ struct CollectDataForFluxes<BoundaryScheme, domain::Tags::InternalDirections<
               all_mortar_data->at(mortar_id).local_insert(
                   temporal_id, std::move(boundary_data_on_mortar));
             });
+
+        if constexpr (db::tag_is_retrievable_v<
```

Comment:
- For my understanding: this `if constexpr` is only needed because this code is shared with the elliptic solver?
- More of a note for me: this will need to be added to `ApplyBoundaryCorrections` once the LTS additions I have there are merged (submitting PR later today or Monday/Tuesday)

Reply:
Yes, that is for the elliptic code.

Reply:
FYI, the elliptic code will stop using this action with the upcoming DG operator. I was waiting for a few of the current preparation PRs before opening the PR for the new DG operator.